planner/core/task.go

```diff
@@ -960,7 +960,20 @@ func (t *copTask) convertToRootTaskImpl(ctx sessionctx.Context) *rootTask {
 		}.Init(ctx, t.tablePlan.SelectBlockOffset())
 		p.PartitionInfo = t.partitionInfo
 		p.stats = t.tablePlan.statsInfo()
-		if needExtraProj {
+
+		// If agg was pushed down, the partial agg was placed on the top of tablePlan, the final agg will be placed
```

Comment:
s/was/is

Reply:
I prefer "was" because when it comes here, the "push down" process has been done.
I slightly changed the comments to try to make it more clear, do you think it's ok?